product_docs/docs/pgpool/4.3/01_installing_and_configuring_the_pgpool-II/x86_amd64/07a_pgpool_ubuntu18_x86.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: "Installing EDB PgPool-II on Ubuntu 18.04 x86"
+navTitle: "Ubuntu 18.04"
+---
+
+To install EDB Pgpool-II on a Debian or Ubuntu host, you must have credentials that allow access to the EDB repository. To request credentials for the repository, visit the [EDB website](https://www.enterprisedb.com/user).
+
+Perform the following steps to install a Debian package using the EDB apt repository.
+
+1.  Assume superuser privileges:
+
+    ```text
```

Comment:
Let's make a reminder to hit these for highlighting/ tagging soon too

Reply:
The generated topics have the proper language (at least the SLES ones do).